assets/training/distillation/components/data_generation/spec.yaml

```diff
@@ -83,8 +83,20 @@ inputs:
 
   enable_chain_of_thought:
     type: string
-    default: "true"
+    default: "false"
     description: Enable Chain of thought for data generation
+  
+  data_generation_task_type:
+    type: string
+    enum:
+      - NLI
```

Comment:
will it better to keep enums unabbreviated?

Reply:
@skasturi Can you comment here.
The unabbreviated terms are very lengthy. For example NLI becomes Natural Language Inference

Reply:
Actual enum can be abbreviated. In the help we need to make sure we are describing what each of them correspondings to.